tendrl/gluster_integration/sds_sync/utilization.py

```diff
@@ -40,7 +40,9 @@ def sync_utilization_details(volumes):
     out = ''
     try:
         out, err = cmd.communicate()
-        if err == '':
+        if err == '' or 'failed to load module nvdimm: '\
```

Comment:
actual error message is '\n (process:2297): WARNING : 17:44:53.361: failed to load module nvdimm: libbd_nvdimm.so.2: cannot open shared object file: No such file or directory\n'